### DISCUSSION TITLE
Why is `path` an optional property?

### DISCUSSION THREAD

**Discussion by jonkoops** [remix-run/react-router#10270] [Q&A] [answered]
As far as I can tell the `path` property is required for all non-index routes. So why is it marked as optional for the [`NonIndexRouteObject`](https://github.com/remix-run/react-router/blob/a5e3ca647d733211599310626366e75b1ee228a7/packages/react-router/lib/context.ts#L38) type?

**Comment by brophdawg11** [comment, marked as answer]
It's optional for use in [pathless layout routes](https://reactrouter.com/en/main/start/concepts#pathless-routes) so you don't have to type `path=""`
